appcues/src/main/java/com/appcues/trait/appcues/ModalTrait.kt

```diff
@@ -28,7 +28,7 @@ internal class ModalTrait(
 
     @Composable
     override fun WrapContent(content: @Composable (hasFixedHeight: Boolean, contentPadding: PaddingValues?) -> Unit) {
-        val windowInfo = rememberWindowInfo()
+        val windowInfo = rememberAppcuesWindowInfo()
```

Comment:
renamed WindowInfo to AppcuesWindowInfo to ensure we dont confuse it with android System WindowInfo class

Reply:
also makes it clear its a custom object in this case since both have same concept behind it